src/tests/Missions.test.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from '../redux/configureStore';
```

Comment:
- [ ] Kindly check this import `../redux/configureStore`, I had issues starting your project because the name of the folder starts with a capital `R`edux and you imported it using a Lowercase `r`edux